frontend/src/Components/Error/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from "react";
+import Button from "plaid-threads/Button";
+import Note from "plaid-threads/Note";
+
+import { ErrorDataItem } from "../../dataUtilities";
+
+import styles from "./index.module.scss";
+
+interface Props {
+  error: ErrorDataItem;
+}
+
+const errorPaths: { [key: string]: string } = {
+  ITEM_ERROR: "item",
+  INSTITUTION_ERROR: "institution",
+  API_ERROR: "api",
+  ASSET_REPORT_ERROR: "assets",
+  BANK_TRANSFER_ERROR: "bank-transfers",
+  INVALID_INPUT: "invalid-input",
+  INVALID_REQUEST: "invalid-request",
+  INVALID_RESULT: "invalid-result",
+  OAUTH_ERROR: "oauth",
+  PAYMENT_ERROR: "payment",
+  RATE_LIMIT_EXCEEDED: "rate-limit-exceeded",
+  RECAPTCHA_ERROR: "recaptcha",
+  SANDBOX_ERROR: "sandbox",
+};
+
+const Error = (props: Props) => {
+  const [path, setPath] = useState("");
+
+  useEffect(() => {
+    const errorType = props.error.error_type!;
+    const errorPath = errorPaths[errorType];
+
+    setPath(
+      `https://plaid.com/docs/errors/${errorPath}/#${props.error.error_code?.toLowerCase()}`
+    );
+  }, [props.error]);
+
+  return (
+    <>
+      <div className={styles.errorTop}></div>
```

Comment:
what is this extra div for? could the border just be added to the container?

Reply:
because the container takes the whole width and the top border should not extend to the edges.